pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/fst/PinotBufferIndexInput.java

```diff
@@ -31,11 +31,11 @@
  */
 public class PinotBufferIndexInput extends IndexInput {
   private final PinotDataBuffer _pinotDataBuffer;
-  private final Long _sliceOffset;
-  private final Long _length;
-  private Long _readPointerOffset;
+  private final long _sliceOffset;
+  private final long _length;
+  private long _readPointerOffset;
 
-  public PinotBufferIndexInput(PinotDataBuffer pinotDataBuffer, Long offset, Long length) {
+  public PinotBufferIndexInput(PinotDataBuffer pinotDataBuffer, long offset, long length) {
```

Comment:
👍 

Looks like this class is only used to read text index. btw, I spotted that at L76 below, `Byte` was used and could be changed to `byte`.